### PR TITLE
Update report.md

### PR DESCRIPTION
**Pull request by Restent**
咕咕咕

### CODE REVIEW

**Review by Cubik65536 — CHANGES_REQUESTED** [review]
「首先；其次」应该使用Firstly，secondly而非First，sencond。

其他的地方仍可提高，不过不影响理解。

**Comment by Restent** [comment]
dalao 帮我吧，我眼困了（


2021年5月31日21:42 来自 ***@***.***:

File: report.md [hunk 1]
```diff
@@ -1,27 +1,28 @@
-# 报告问题的正确姿势
+# How to request question
```
Comment (Cubik65536) [inline report.md:1]: 根据文件名称提出更改建议：
How to report a question

File: report.md [hunk 2]
```diff
@@ -1,27 +1,28 @@
...
 
-任何事物都不可能尽善尽美，LittleSkin 也是如此。
+Everything are not possible to perfect, LittleSkin is also like this.
```
Comment (Cubik65536) [inline report.md:3]: 提出建议，即供参考：

Nothing is perfect, nor LittleSkin.

File: report.md [hunk 3]
```diff
@@ -1,27 +1,28 @@
...
 
-如果你在使用 LittleSkin 的过程中遇到了任何问题（包括但不限于材质无法显示、网站出现 Bug 等），不要气急败坏，请充分理解下述内容后正确提问。
+If you have any problem that you use LittleSkin (including but not limited to: texture can't display, website have bugs etc.), don't utterly discomfited， please understand the following content fully, and then ask question correctly.
```
Comment (Cubik65536) [inline report.md:5]: > ...that you use LittleSkin...

...**while** you use LittleSkin...

> including but not limited to

非常好

> ... texture can't display, website have bugs etc.

- etc前面要写逗号 `..., etc.`

- 更改建议：textures aren't showing, bugs on the website, etc.

> 气急败坏 -》utterly discomfited

翻译确实没错，但是有点怪异，先这么写吧

> please understand the following content fully

建议：please fully understand the following content

File: report.md [hunk 4]
```diff
@@ -1,27 +1,28 @@
...
 
-## 遇到问题后的第一反应
+## The first reaction when having a problem
```
Comment (Cubik65536) [inline report.md:7]: > when

更改建议：while

File: report.md [hunk 5]
```diff
@@ -1,27 +1,28 @@
...
 
 > DON'T PANIC.
 
-首先，你应该先检查一下是不是你的问题。大部分情况下，LittleSkin 是可以正常运行的。如果你遇到了错误，请先确定这是不是你自己的锅：网络错误、操作错误、甚至是你自己的浏览器出问题都可能导致你无法正常使用 LittleSkin。所以，在提问之前，请先以自己的经验判断一下，这是不是你的问题。
+First, you should check if this is your problem at first. At many times, LittleSkin can run normally. If you get an error, please make sure that this isn't your problem at first: Network error, Operating error, even your browser error might can't let you use LittleSkin normally. So, before asking questions, please judgment with your experience, this is your problem or not.
```
Comment (Cubik65536) [inline report.md:11]: > First,

**Firstly, **

> if this is your problem

更改建议：if it's your problem

> At many times,

更改建议： For most of the time,

> Operating error

更改建议：wrong operation

> even your browser error might can't let you use LittleSkin normally

更改建议：even your browser's error can makes you run into problem while using LittleSkin 

> please judgment with

please judge with

File: report.md [hunk 6]
```diff
@@ -1,27 +1,28 @@
...
 
-其次，你应该先阅读一遍 [常见问题解答](/faq.html)。很多情况下你遇到的问题也曾经困扰过很多人，并且已经被总结出了解决方案。这种情况下，直接查阅 FAQ 能更快地解决你的问题。
+Second, you should read [FAQ](/faq.html) at first. At many times, your problem might also trouble many people, and these problem are already have solution. At this situation, read FAQ directly can solve your problem fastly.
```
Comment (Cubik65536) [inline report.md:13]: > Second,

Secondly, 

> at first

删掉它，secondly和first相互冲突

> At many time, 

更改建议：Sometimes,

> your problem might also trouble many people

更改建议：your problem might troubled many other people

> and these problem are already have solution

更改建议：
- and these problems have already a solution
- and there's already a solution for these problems

> At this situation

In this situation

> read FAQ

read the FAQ

> fastly

quickly（fastly不是个单词啊！）

File: report.md [hunk 7]
```diff
@@ -1,27 +1,28 @@
...
 
-如果你确定你没有问题，并且你遇到的问题不在 FAQ 中，你可以着手准备报告问题了。
+If you sure that you don't have any problem, and your problem isn't in FAQ, you can prepare to report problem.
```
Comment (Cubik65536) [inline report.md:15]: > you don't have any problem

这么写是指：你没有遇到任何问题

更改建议：If you're sure that it's not your problem

> in FAQ

in the FAQ

> report problem

report the problem

File: report.md [hunk 8]
```diff
@@ -1,27 +1,28 @@
...
 
-:::tip 推荐阅读：<a href="https://github.com/ryanhanwu/How-To-Ask-Questions-The-Smart-Way/blob/master/README-zh_CN.md" target="_blank">《提问的智慧》</a>
+:::tip We recommend you to read：<a href="http://www.catb.org/~esr/faqs/smart-questions.html" target="_blank">How To Ask Questions The Smart Way</a>
 
-充分理解这篇文章可以帮助你更好地提问并更快地收到回复和正确的解答（不论何时何地）。但请注意，《提问的智慧》的创作团队仅是提供这一篇文章来指导你如何正确提问，他们并不提供针对任何问题的任何技术支持。**不要做白痴。**
+Understang this article fully can help you ask questions better, and also receive the reply and the right answer fastly. But please attention, the team of How To Ask Questions The Smart Way is only provide this article to guide you how to ask questions correctly, they don't provide any technical support of any problem. **Don't be an idiot.**
```
Comment (Cubik65536) [inline report.md:19]: > Understang this article fully

Fully understand this article

> fastly

**quickly** !!!

> But please attention

But please **pay** attention

File: report.md [hunk 9]
```diff
@@ -1,27 +1,28 @@
...
 :::
 
-## 运营团队不是神
+## Operation team isn't god
```
Comment (Cubik65536) [inline report.md:22]: 更改建议：Operating staffs aren't god

File: report.md [hunk 10]
```diff
@@ -1,27 +1,28 @@
...
 
 看起来你们大多数人好像都不知道的样子？
+Looks like many of "you" are
```
Comment (Cubik65536) [inline report.md:25]: 更改建议：It seems to me most of you don't know it.

**Review by Cubik65536 — CHANGES_REQUESTED** [review]
整体内容无误，但仍有部分语法问题

我已留下部分改进建议，希望能够帮到你

Best regards,

**Comment by Cubik65536** [comment]
已基于该版翻译进行优化并补齐其他翻译，查看 #1

File: report.md (hunk 3 above)
Comment (jinzhijie) [reply report.md:5]: ① while 引导的时态状语从句应该用现在时吧？所以不应该是 `... while you are using LittleSkin ...` 么？

③ 可以改成类似于 wrong textures 这种，至少得是个名词

**Pull request closed without merging**